core/build_vars.go

```diff
@@ -64,3 +64,9 @@ func BuildInfo() string {
 
 	return b.String()
 }
+
+// UserAgent returns a string that can be used as HTTP user agent, containing the version of the node (e.g. nuts-node-refimpl/5.0.0)
+func UserAgent() string {
+	// Remove leading "v" from version to keep it standard for user agent strings.
+	return "nuts-node-refimpl/" + strings.TrimPrefix(Version(), "v")
```

Comment:
If the node is not running a tagged version, `Version()` returns the branch name. Is this okay?

Reply:
if so, what about trimming of branch names starting with `vdr/...` or `vcr/...`

Reply:
Just return 0 if it isn't a version (so it must start with `v` and following a digit)?

Reply:
Or use `GitVersion`? This makes it easy to differentiate between official releases and custom builds.